samples/bluetooth/peripheral_iso/src/main.c

```diff
@@ -0,0 +1,150 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/printk.h>
+#include <sys/byteorder.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/iso.h>
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+};
+
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	if (err) {
+		printk("Failed to connect to %s (%u)\n", addr, err);
+		return;
+	}
+
+	printk("Connected %s\n", addr);
+}
+
+static void disconnected(struct bt_conn *conn, uint8_t reason)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	printk("Disconnected from %s (reason 0x%02x)\n", addr, reason);
+}
+
+static struct bt_conn_cb conn_callbacks = {
+	.connected = connected,
+	.disconnected = disconnected,
+};
+
+static void iso_recv(struct bt_iso_chan *chan, struct net_buf *buf)
+{
+	char data_str[35];
+	size_t len = 0;
+	printk("Incoming data channel %p len %u\n", chan, buf->len);
+
+	/* Print data as d_0 d_1 d_2 ... d_(n-2) d_(n-1) d_(n)
+	 * to show the 3 first and 3 last octets
+	 */
+	len += bin2hex(buf->data, MIN(3, buf->len), data_str + len, sizeof(data_str) - len);
+	if (buf->len > 3) {
+		if (buf->len > 6) {
+			static char dots[] = " ... ";
+			strcat(&data_str[len], dots);
```

Comment:
Can be changed to 
```C
strcat(&data_str[len], " ... ");
len += 5
```

Reply:
I'd rather keep it as is, such that we can use strlen in case we want to change the string (I actually just also changed it from " ... " to "..." to better match the format of `bin2hex`